docs/en/observability/profiling-self-managed.asciidoc

```diff
@@ -0,0 +1,840 @@
+[[profiling-self-managed]]
+= Run Universal Profiling on self-hosted Elastic stack
+
+++++
+<titleabbrev>Self-hosted infrastructure</titleabbrev>
+++++
+
+beta::[]
+
+Here you'll find information on running Universal Profiling when hosting the Elastic stack on your own infrastructure.
+
+Universal Profiling consists of a client part (the host-agent) and a server part (the backend). This documentation focuses on the server part.
+
+[discrete]
+[[profiling-self-managed-supported-platforms]]
+== Prerequisites
+
+IMPORTANT: Running Universal Profiling on self hosted infrastructure requires an Elastic **Enterprise license**.
```

Comment:
Is this reflected on https://www.elastic.co/subscriptions? For legal reasons, we don't name the license level required in the documentation and instead link to the source of truth (the subscriptions page). Here's an example from the SLO docs:

> To create and manage SLOs, you need an {subscriptions}[appropriate license].

Can we do the same here? Will UP be added to the subscriptions page when 8.12 releaes?

Reply:
I summon @iogbole on this one as I have no clue about the other page.
I am only aware that for on-prem we need Enterprise... we have a check in the code to enforce that.

@bmorelli25 can suggest what should be the appropriate format here?
I guess

```suggestion
IMPORTANT: Running Universal Profiling on self hosted infrastructure requires an Elastic **{subscriptions}[Enterprise license]**.
```

Reply:
I've moved this line to the top of the page [here](https://github.com/elastic/observability-docs/pull/3379/files#diff-e88124970af8998ed20bdedbec48f775f64ca3f36a214b701b664541d982bcfeR10) to be consistent with the rest of the obs docs. 

If we are able to get the Profiling subscription info on the subscriptions page, we wouldn't want to explicitly say what kind of license users need in the docs. With this in mind, I phrased it this way:

> IMPORTANT: To run Universal Profiling on self-hosted Elastic stack, you need an {subscriptions}[appropriate license].

If we aren't able to get the info on the subscriptions page, maybe we can rephrase to be specific in this instance.